website/docs/r/user.html.markdown

```diff
@@ -26,6 +26,16 @@ resource "mysql_user" "jdoe" {
 }
 ```
 
+```hcl
```

Comment:
Could a heading be added here denoting that this example is a an example using an authentication plugin?